PhysicsTools/NanoAOD/python/jetsAK8_cff.py

```diff
@@ -199,15 +234,9 @@ def nanoAOD_addDeepInfoAK8(process, addDeepBTag, addDeepBoostedJet, addDeepDoubl
     nanoAOD_addDeepDoubleX_switch = cms.untracked.bool(False),
     nanoAOD_addDeepDoubleXV2_switch = cms.untracked.bool(False),
     nanoAOD_addParticleNet_switch = cms.untracked.bool(False),
-    nanoAOD_addParticleNetMass_switch = cms.untracked.bool(False),
     jecPayload = cms.untracked.string('AK8PFPuppi')
 )
 
-# for 106Xv2: only needs to run ParticleNet Mass regression; The rest are already in MiniAOD
```

Comment:
I think that now that we still store the legacy ParticleNet for 106Xv2 samples, this and the above changes need to be reverted to rerun the mass regression for these samples?